ts/DataGrid/Table/Table.ts

```diff
@@ -396,7 +414,9 @@ class Table {
      */
     public destroy(): void {
         this.tbodyElement.removeEventListener('focus', this.onTBodyFocus);
-        this.tbodyElement.removeEventListener('scroll', this.onScroll);
+        if (this.dataGrid.options?.rendering?.rows?.virtualization) {
+            this.tbodyElement.addEventListener('scroll', this.onScroll);
```

Comment:
Do we really need this in if? Anyway:

```suggestion
            this.tbodyElement.removeEventListener('scroll', this.onScroll);
```